recipes/winmd/all/conanfile.py

```diff
@@ -0,0 +1,79 @@
+import os
+
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import copy, get
+from conan.tools.layout import basic_layout
+from conan.tools.scm import Version
+from conan.tools.microsoft import is_msvc
+
+required_conan_version = ">=1.52.0"
+
+
+class WinMDConan(ConanFile):
+    name = "winmd"
+    description = "C++ winmd parser"
+    # Use short name only, conform to SPDX License List: https://spdx.org/licenses/
+    # In case not listed there, use "LicenseRef-<license-file-name>"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/microsoft/winmd"
+    # no "conan" and project name in topics. Use topics from the upstream listed on GH
+    # Keep 'hearder-only' as topic
+    topics = ("native", "C++", "WinRT", "WinMD")
+    settings = "os", "arch", "compiler", "build_type"  # even for header only
+    no_copy_source = True  # do not copy sources to build folder for header only projects, unless, need to apply patches
+
+    @property
+    def _min_cppstd(self):
+        return 17
+
+    # in case the project requires C++14/17/20/... the minimum compiler version should be listed
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "Visual Studio": "16",
```

Comment:
There's a new helper for the migrations with the new msvc compiler

https://github.com/conan-io/conan-center-index/blob/75655c82d50b3c6e57c4cf79a42b277b181083d4/docs/package_templates/cmake_package/all/conanfile.py#L80